src/lib/components/NewLayeredMap/NewLayeredMap.jsx

```diff
@@ -372,6 +372,11 @@ NewLayeredMap.propTypes = {
      * Map coordinates of a mouse click
      */
     click_position: PropTypes.array,
+
+    /**
+     * Allows to choose between replacing the layers or updating them
+     */
+    layerChangeMethod : PropTypes.string,
```

Comment:
Can we maybe call this one _updateStrategy_ or something? 🤔

Reply:
I'm working on this, and I use the name updateMode, but I thought I'd just change it when I merge this into my branch